src/CPU/ctxswitch.cpp

```diff
@@ -56,7 +56,7 @@ double total_pass() {
             exit(1);
         } else if (chid == 0) {
             char buf;
-            use_cores(vector<int> {0});
+            use_cores(vector<int> {1});
             vector<int> f(footprint_vector_len, 1);
```

Comment:
Can we remove this if not used?

Reply:
it is used, if threads are allowed to use different cpu's we can't guarantee a context switch

Reply:
My concern was about line 60.

Reply:
that is the cache footprint, this simulates a real process since now it has a realistic associated memory footprint. see the Imbench paper for details. Context switch time grows as we increase the footprint and have to use the lower levels of memory hierarchy